quizup/src/Routes/BrowseCoding/BrowseCoding.css

```diff
@@ -0,0 +1,38 @@
+.BrowseCoding-section {
+  display: flex;
+  column-gap: 25rem;
+  align-items: center;
+  justify-content: center;
+  margin-top: 15rem;
+}
+.react-quiz,
+.js-quiz {
+  width: 20%;
+  cursor: pointer;
+  box-shadow: rgba(50, 50, 93, 0.25) 0px 50px 100px -20px,
+    rgba(0, 0, 0, 0.3) 0px 30px 60px -30px,
+    rgba(10, 37, 64, 0.35) 0px -2px 6px 0px inset;
+  padding-top: 1rem;
+}
+
+.image-top img {
+  width: 100%;
+  height: 10%;
+}
+
+.text-bottom {
+  padding: 2rem;
+}
+
+.play-btn {
+  background-color: #6200ee;
+  color: white;
+  font-weight: 900;
+  width: 40%;
+  padding-top: 0.5rem;
```

Comment:
units below 1rem could be given in pixels unless you want your padding to increase as the user increases root font size.